webapp/cypress/integration/common/global_reuse.js

```diff
@@ -0,0 +1,19 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps'
```

Comment:
przyrostek reuse do wywalenia we wszystkich testach

Reply:
zmieniłem